### DISCUSSION TITLE
Folder icon thumbnails?

### DISCUSSION THREAD

**Discussion by kilrah** [valinet/ExplorerPatcher#427] [Ideas]
Would it be within the realm of possibilities for this tool to restore the content thumbnails on folder icons?

**Comment by valinet** [comment]
I usually do not ignore stuff that I know how to do, I implement it and release right away. 

Everything in this world is doable, one just needs the time, motivation and (possibility to acquire) the knowledge to do so, if I may speak in the same realm as your question. Of course, if studied, this can be achieved, as can everything else. 

I'd also love to have this contributed as well. I am not using this feature at all, so I have very few experience regarding its inner workings. I don't currently have a starting point for looking into it, and honestly, I am quite tired of this software, I have been doing work on this non stop for the past couple of months, it's starting to look a bit ridiculous that one guy there and some guys over there and so on have to fix up a billion dollars company's shipping product. I mean, yeah, I enjoy the technical challenge and the learning part, but it gets tiring, you sometimes feel the need for a change, y'know...

Also, I am quite surprised, honestly, considering the amount of user base Windows has, that besides the big dogs like Stardock, and the smaller but still significant players like StartAllBack (which has been doing a tremendous job for a long time), not many else is interested in solving these problems. Not even Microsoft. Not to mention solving them in a transparent manner, that's like another league already, but rather providing a fix at all. Looking at the backlash, one would have expected the developer involvement to be much greater. In reality, there's a few guys here and there and that's it. I can't believe that all knowledgeable users have moved to other platforms, so I cannot understand the lack of involvement from other folks. Programs don't write themselves, and the demands and expectations keep growing, while the people willing to put up the effort is decreasing day after day, not to mention a day still has 24 hours. The involvement with this project is lackluster to say the least. Even in non-developer areas, at most one's interested in their area and that's it, almost no one contributes to help others especially if it doesn't directly benefit them. If I were to apply the same principle to EP, the software would have been mostly done months ago, or have a whole different perspective or feature set than it has now. I am quite disappointed to be honest, I was expecting more. Or rather, not expecting, but hoped for. But yeah, hopes and reality are 2 totally different things.

This is not a reply to you directly, I just felt like writing that down at this moment. I understand this request, specifically, as I always say, if I find something out, I never hesitate to look it up and try to bring it to a usable state and deliver it as an update. That's what I have been constantly doing since this project started.

Thanks

**Comment by plofhaan** [comment]
https://github.com/hahagu/WindowsThumbnailGenerator
